packages/repocop/src/evaluation/repository.ts

```diff
@@ -506,6 +507,7 @@ export function evaluateRepositories(
 		const isMainBranchPredicate = (x: Tag) =>
 			x.key === 'branch' && (x.value === 'main' || x.value === 'master');
 
+		// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion -- to allow ts version bump
```

Comment:
```suggestion
		// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion -- to allow ts version bump
```
just remove the `as string[]` assertion at the end of this chain